From 0668f7e1af44a6eddc4cd716a9b61fab4e10de7b Mon Sep 17 00:00:00 2001
From: Jani Hautakangas <jani.hautakangas@lge.com>
Date: Wed, 4 Apr 2018 21:10:09 +0300
Subject: GLIBC fix: HOST_NAME_MAX requires include <limits.h>

Compiling fails with glibc that has been built with 3.2 kernel support.
HOST_NAME_MAX is not declared in scope in cloud_policy_utils.c

Change-Id: I1d70125b1101176cb08f317affbc760e65a726e5

diff --git a/components/policy/core/common/cloud/cloud_policy_util.cc b/components/policy/core/common/cloud/cloud_policy_util.cc
index b0d758c3f243..0395270e2c65 100644
--- a/components/policy/core/common/cloud/cloud_policy_util.cc
+++ b/components/policy/core/common/cloud/cloud_policy_util.cc
@@ -16,6 +16,7 @@
 #include <sys/sysctl.h>
 #endif
 
+#include <limits.h>
 #include <utility>
 
 #include "base/strings/utf_string_conversions.h"
-- 
2.14.1

